src/databricks/labs/mcp/servers/unity_catalog/app.py

```diff
@@ -0,0 +1,65 @@
+from mcp.server import NotificationOptions, Server
```

Comment:
Nice, any reason not to just replace the existing stdio-based server implementation with this one? We can do it in a follow-up PR, but just curious

Reply:
Eventually seems like we'd just have one streamable-HTTP-based implementation (once more clients support streamable HTTP)

Reply:
Also, depending on how much work it is, it's actually now possible to use streamable HTTP to write the server, as of four days ago! https://github.com/modelcontextprotocol/python-sdk/releases/tag/v1.8.0

Reply:
I'll update my version then, should work easily.

Reply:
Done, updated!